auto_nag/scripts/duplicate_copy_metadata.py

```diff
@@ -0,0 +1,172 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import List
+
+from libmozdata.bugzilla import Bugzilla
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+
+
+class DuplicateCopyMetadata(BzCleaner):
+    def description(self):
+        return "Copied fields from duplicate bugs"
+
+    def handle_bug(self, bug, data):
+        bugid = str(bug["id"])
+        data[bugid] = bug
+
+        return bug
+
+    def get_bugs(self, date="today", bug_ids=[], chunk_size=None):
+        dup_bugs = super().get_bugs(date, bug_ids, chunk_size)
+
+        original_bug_ids = {bug["dupe_of"] for bug in dup_bugs.values()}
+        original_bugs = {}
+
+        Bugzilla(
+            {
+                "id": original_bug_ids,
+                "bug_status": "__open__",
+            },
+            include_fields=[
+                "id",
+                "summary",
+                "whiteboard",
+                "keywords",
+                "duplicates",
+            ],
+            bughandler=self.handle_bug,
+            bugdata=original_bugs,
+        ).wait()
+
+        results = {}
+        for bug_id, bug in original_bugs.items():
+            copied_fields = {}
+            for dup_bug_id in bug["duplicates"]:
+                dup_bug = dup_bugs.get(str(dup_bug_id))
+                if not dup_bug:
+                    continue
+
+                # Keywords: copy the `access` keyword from duplicates
+                if "access" not in bug["keywords"] and "access" in dup_bug["keywords"]:
+                    if "keywords" not in copied_fields:
+                        copied_fields["keywords"] = {
+                            "from": [dup_bug["id"]],
+                            "value": "access",
+                        }
+                    else:
+                        copied_fields["keywords"]["from"].append(dup_bug["id"])
+
+                # Whiteboard: copy the `access-s*` whiteboard rating from duplicates
+                if (
+                    "access-s" not in bug["whiteboard"]
```

Comment:
We should also copy it if the duplicate has a higher accessibility severity than the bug it was duplicated to

Reply:
I prefer not to overwrite what is already in the bug. For example, in the severity case (i.e., the `severity_higher_dups` tool) we set a needinfo instead of overwriting the severity. We could do something similar thing for the accessibility (as part of this tool or in a separate one).

Reply:
OK, works for me.